demo/atom/card/demo/index.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable react/prop-types, no-unused-vars, no-console */
+
+import React from 'react'
+
+import AtomImage from '@s-ui/react-atom-image'
+import AtomCard from '../../../../components/atom/card/src'
+import LayoutMediaQuery from '@s-ui/react-layout-media-query'
```

Comment:
you could maybe add a responsive card that switches from vertical to horizontal

Reply:
Done!

Reply:
How that will works with the SSR ?!